api/client/container/wait.go

```diff
@@ -28,7 +28,6 @@ func NewWaitCommand(dockerCli *client.DockerCli) *cobra.Command {
 			return runWait(dockerCli, &opts)
 		},
 	}
-	cmd.SetFlagErrorFunc(flagErrorFunc)
```

Comment:
I think the definition for  `func flagErrorFunc()` can be deleted as well? That way we can be sure it's not use anywhere else.